src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="body">
+    <main class="container">
+      <div class="container__element b1">1</div>
+      <div class="container__element b2">2</div>
+      <div class="container__element b3">3</div>
+      <div class="container__element b4">4</div>
+      <div class="container__element b5">5</div>
+      <div class="container__element b6">6</div>
```

Comment:
what is this b1, b2, b3, ... blocks? if you tried to make modificators, than you shouldn't, use nth-child instead

Reply:
⚠️

Reply:
pay attention to comments, you haven't fixed them at all